web-portal/components/admin-venue-edit/VenueCard.vue

```diff
@@ -153,6 +199,27 @@
 
         this.$store.dispatch(DELETE_VENUE, { id, idToken })
       },
+      onFindUsingMapLinksClick: function () {
+        this.$data.fetchingGpsCoordinates = true
+        this.$data.fetchingGpsCoordinatesError = null
```

Comment:
Nitpick here but there's no reason for the `$data` part -- should be just `this.fetchingGpsCoordinates = true` etc

Reply:
ok cool good to know, will update